expect.go

```diff
@@ -1096,6 +1099,15 @@ func (e *GExpect) Read(p []byte) (nr int, err error) {
 	return e.out.Read(p)
 }
 
+// Return the unmatched suffix back to the buffer so that it can be matched again
```

Comment:
Comments should be of the format // ReturnUnmatchedSuffix ...

Reply:
Will be fixed in the next iteration.